corehq/toggles.py

```diff
@@ -631,6 +634,7 @@ def _ensure_valid_randomness(randomness):
         "compatability for old projects.  We are actively trying to remove "
         "projects from this list."
     ),
+    always_enabled={'icds-cas'}
```

Comment:
i am not sure if this should be set as "always_enabled". ICDS is currently using it but i think they should be using FLAT location fixture eventually and hence then disable this from the FF

Reply:
having this here doesn't mean we can't ever switch it off - it just short circuits the cache / DB lookup. When we want to switch it off we can change this to `always_disabled`. I don't think we'll need to do it dynamically though if that's the case we can remove it then.

Reply:
ya i meant, we would need a code change instead of just switching it off from the feature flag listing.